utilities_common/netstat.py

```diff
@@ -23,10 +23,10 @@ def ns_brate(newstr, oldstr, delta):
         return STATUS_NA
     else:
         rate = int(ns_diff(newstr, oldstr).replace(',',''))/delta
-        if rate > 1024*1024*10:
-            rate = "{:.2f}".format(rate/1024/1024)+' MB'
-        elif rate > 1024*10:
-            rate = "{:.2f}".format(rate/1024)+' KB'
+        if rate > 1000*1000*10:
```

Comment:
It should not be 1000. 
1024 is the correct value.

Reply:
For throughput calculation against the link speed, factor has to be 1000 and not 1024.
https://my.ciena.com/CienaPortal/s/question/0D50z00005JK3dQCAT/ethernet-packet-rate-and-throughput-calculations

Reply:
the reference mentions to convert Link speed in 1000's instead of 1024.
Here we are converting frame size, not the speed